src/atoms/Heading/Heading.web.js

```diff
@@ -56,18 +57,20 @@ const Heading = ({ size, color, children, maxLines, weight }) => {
   const theme = useTheme();
 
   return (
-    <Text
-      as={headingLevel[size]}
-      size={styles.fontSize({ theme, size })}
-      color={color}
-      maxLines={maxLines}
-      _weight={weight}
-      _lineHeight={styles.lineHeight({ theme, size, weight })}
-      _letterSpacing="small"
-      {...automation('ds-heading')}
-    >
-      {children}
-    </Text>
+    <Space margin={[0]}>
```

Comment:
```suggestion
    <Space margin={0}>
```

Else it'll just apply apply margin-top:0 with array.

Reply:
Wut? 😮 Didn't know we have done this implemention.

Reply:
No my bad.